amplitude_python_sdk/v2/client.py

```diff
@@ -81,3 +89,75 @@ def batch_upload_events(
         """  # pylint: disable=line-too-long
         req_data = self._construct_event_request_data(events, options)
         return self._log_events_internal(routes.BATCH_API, req_data, timeout)
+
+    @staticmethod
+    def _construct_chart_request_data(
```

Comment:
One suggestion - move these helper methods into a separate file, this way the main client file stays clean and readable.

Reply:
This helper ended up being unnecessary. Removed